### PR TITLE
Initial work on saving and visiting tokens

### PR DESCRIPTION
**Pull request by openwarrior**
Token : A model for token to be saved
TokensAdapter : Transforms a list of Tokens to a human-readable interpretation.
TokensFragment : To be added in container to present data.

'White Plus' icons to be added on Floating Action Bar

### CODE REVIEW

File: src/main/java/org/hawkular/client/android/backend/model/Token.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        @Override public Token createFromParcel(Parcel parcel) {
+            return new Token(parcel);
+        }
+
+        @Override
+        public Token[] newArray(int size) {
+            return new Token[size];
+        }
+    };
+    @RecordId
```
Comment (pilhuhn) [inline src/main/java/org/hawkular/client/android/backend/model/Token.java:38]: Leave an empty line before this

**Comment by pilhuhn** [comment]
Thanks @anuj1708

File: src/main/java/org/hawkular/client/android/backend/model/Token.java (hunk 1 above)
Comment (openwarrior) [inline src/main/java/org/hawkular/client/android/backend/model/Token.java:38]: Added in coming PR :)

**Comment by openwarrior** [comment]
Thanks for your support @pilhuhn

**Pull request merged**